README.md

```diff
@@ -6,6 +6,28 @@ go-ipfix is an IPFIX library that can be used to implement an IPFIX exporter, wh
 ## Try it out
 This IPFIX library can be used to build an exporter. Please check out the [exporter tests](https://github.com/vmware/go-ipfix/blob/main/pkg/exporter/process_test.go) to get an idea on how to build exporter on top of TCP and UDP transport protocols given a IPFIX collector.
 
+### Deploy stand alone IPFIX collector
+To deploy a standalone go-ipfix collector, which is used to decode and log the IPFIX records, you can use:
+```
+cd <directory containing this README file>/build/yamls
+kubectl apply -f ./ipfix-collector.yaml
```

Comment:
We can follow the way antrea does [here](https://github.com/vmware-tanzu/antrea/blob/main/docs/getting-started.md#installation): specify the release manifest here and if users want to customize the protocol etc., they can download the folder from main branch.

Reply:
Absolutely. The only concern I had before is that users will not be able to find any manifests under "releases assets" since we do not have the manifest for previous release. But I agree we can add the guide at this time for future need. Thanks!